data/hails.txt

```diff
@@ -310,6 +316,7 @@ phrase "friendly civilian"
 		"everyone's"
 		"my crew is"
 		"some of my friends are"
+		"the older generation is"
```

Comment:
"I can't believe the older generation is following that new group"

Somehow sounds odd